vercel.json

```diff
@@ -1086,6 +951,13 @@
         { "source": "/docs/batch-exports/s3", "destination": "/docs/cdp/batch-exports/s3" },
         { "source": "/docs/batch-exports/snowflake", "destination": "/docs/cdp/batch-exports/snowflake" },
         { "source": "/blog/startup-golioth", "destination": "/spotlight/startup-golioth" },
-        { "source": "/blog/startup-unified", "destination": "/spotlight/startup-unified" }
+        { "source": "/blog/startup-unified", "destination": "/spotlight/startup-unified" },
+        { "source": "/blog/startup-inlang", "destination": "/spotlight/startup-inlang" }
+    ],
+    "rewrites": [
+        { "source": "/community/profiles/:path*", "destination": "/community/profiles/[id]/index.html" },
+        { "source": "/next-steps/(.*)", "destination": "/next-steps/index.html" },
+        { "source": "/questions/:path((?!topic).*)", "destination": "/questions/[permalink]/index.html" },
+        { "source": "/careers/(.*)", "destination": "/careers" }
     ]
```

Comment:
To help prevent this from happening again, can we move this section to the top? My thinking is that someone adding a redirect is likely to scroll to the bottom and add it. So let's prevent accidentally adding here again

also, let's add a comment
```suggestion
    // CAUTION: Rewrites are different from redirects. Only add a rewrite if you're certain that's what you want and not a redirect.
    "rewrites": [
        { "source": "/community/profiles/:path*", "destination": "/community/profiles/[id]/index.html" },
        { "source": "/next-steps/(.*)", "destination": "/next-steps/index.html" },
        { "source": "/questions/:path((?!topic).*)", "destination": "/questions/[permalink]/index.html" },
        { "source": "/careers/(.*)", "destination": "/careers" }
    ]
```